vllm/engine/arg_utils.py

```diff
@@ -1210,6 +1213,54 @@ def create_engine_config(self) -> VllmConfig:
         )
 
 
+@dataclass
+class V1EngineArgs(EngineArgs):
```

Comment:
I would suggest putting `V1` at the end instead of the beginning, but that's just my preference.

```suggestion
class EngineArgsV1(EngineArgs):
```